index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>🍔 도 와 줘 햄 식 ! 🍟</h1>
+    </header>
+    <div class="container">
```

Comment:
```suggestion
    <main class="container">
```
body의 주요콘텐츠는 main테그를 이용하는 게 좋아요!